script/build.js

```diff
@@ -101,7 +116,11 @@ if (options.buildtype === 'production') {
   ignoreList.push('profile/*');
   ignoreList.push('auth/*');
   ignoreList.push('education/apply-for-education-benefits-new');
+} else if (options.buildtype === 'staging') {
+  ignoreList.push('education/apply-for-education-benefits/application.md');
```

Comment:
I think we don't need to add these education files to the `ignoreList` for staging, since we want them exposed on staging for testing. They're in the list for production, so I think we're good there.